### PR TITLE
Labs 31

### PR DESCRIPTION
**Pull request by michael-maton**
# Description
Fixes # (issue)

## Type of change
Please delete options that are not relevant.
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Change Status
- [ ] Complete, tested, ready to review and merge
- [ ] Complete, but not tested (may need new tests)
- [ ] Incomplete/work-in-progress, PR is for discussion/feedback

# How Has This Been Tested?
- [ ] Test A
- [ ] Test B

# Checklist
- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] My code has been reviewed by at least one peer
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] There are no merge conflicts

### CODE REVIEW

File: api/dsService/dsUtil.js [hunk 1]
```diff
@@ -64,35 +63,159 @@ function dsInitialFetch() {
...
+        incomingNewIncidents.push(newIncident);
+      });
 
-  if (mm < 10) {
-    mm = `0${mm}`;
-  }
+      const chunkSize = 30;
+      knex
+        .batchInsert('incidents', incomingNewIncidents, chunkSize)
+        .then((batchResponse) => {
```
Comment (Christopher-Barrett) [inline api/dsService/dsUtil.js:97]: Implement testing for batchResponse

**Review by Christopher-Barrett — APPROVED** [review]

File: api/app.js [hunk 2]
```diff
@@ -24,10 +27,15 @@ const swaggerUIOptions = {
...
 const profileRouter = require('./profile/profileRouter');
 const incidentsRouter = require('./incidents/incidentsRouter');
+const twitterIncidentsRouter = require('./twitter_incidents/twitterIncidentsRouter');
 const dataRouter = require('./util/dataRouter');
 
 //###[ Models ]###
-const { dsFetch } = require('./dsService/dsUtil');
+const {
+  dsUpdateFetch,
+  getLastId,
```
Comment (BrettMcAdams) [inline api/app.js:36]: will find assigned value

File: api/dsService/dsUtil.js [hunk 3]
```diff
@@ -1,12 +1,14 @@
 const axios = require('axios');
 const knex = require('../../data/db-config');
 const Incidents = require('../incidents/incidentsModel');
```
Comment (BrettMcAdams) [inline api/dsService/dsUtil.js:3]: will assign value

**Review by BrettMcAdams — COMMENTED** [review]
Let's hope for the best!

**Review by BrettMcAdams — APPROVED** [review]

**Pull request merged**